hasher-matcher-actioner/hmalib/common/s3_adapters.py

```diff
@@ -114,6 +113,7 @@ def _parse_file(self, file_name: str, data_file: S3FileT) -> t.List[HashRowT]:
             codecs.getreader("utf-8")(data_file["Body"]),
             fieldnames=self.indicator_type_file_columns,
         )
+        privacy_group = file_name.split("/")[-1].split(".")[0]
```

Comment:
Can you add a comment where ever this file_name is created so that future refactors do not accidentally break this?

Reply:
Done. (also added a TODO because where it is created has the extension hardcoded for some reason...)